src/components/key-value-table/key-value-table.scss

```
@@ -71,4 +71,16 @@ $ns: "key-value-table";
       font-family: $font-main;
     }
   }
+  &.#{$ns}--muted {
+    tr {
+      border-color: $gray1;
+    }
+    tr:first-child {
+      border-top: none;
+    }
+
+    tr:last-child {
+      border-bottom: none;
+    }
+  }
```

(non-blocking): Might as well lean into the parent selector as we're using it already :)

```suggestion
  &.#{$ns}--muted {
    tr {
      border-color: $gray1;
      
      &:first-child {
        border-top: none;
      }
      
      &:last-child {
        border-bottom: none;
      }
    }
  }
```